.gitignore

```diff
@@ -47,17 +47,21 @@ src/docs/
 src/ets_lru/
 src/fauxton/
 src/folsom/
+src/hqueue/
+src/hyper/
 src/ibrowse/
 src/ioq/
 src/jiffy/
+src/ken/
 src/khash/
 src/meck/
 src/mochiweb/
 src/oauth/
+src/proper/
```

Comment:
Wonder where we use proper? That's GPL licensed. We had specifically moved from PropEr to Triq before to avoid including GPL-ed license software in ASF CouchDB even in source form.

Reply:
ioq and hqueue use it. Should we switch them to triq?

Reply:
If possible, but it might difficult based on reading comments in IRC.

Another option is to switch to using proper everywhere but make such that proper is an optional dependency (like search or hasting) and run it only if present. In the CI environment and locally on dev machine we'd ensure to make it available but when making a release it won't be.

Reply:
Btw, I didn't mean for my comment to be blocking for this particular PR. Sorry if it seemed that way. I'm +1 to merge it. Proper is an actual dependency and we can solve how to handle that issue separately.

Reply:
FYI any *build* dependency that is GPL is fine - such as gcc - we just can't *ship* GPL code.

Since PropER is just a testing tool, I think we're fine here.

Reply:
Technically we don't use proper anymore? We changed the strategy on ioq and I couldn't find any uses of it in other parts of the code.

It would still be nice to switch to using proper instead of Triq. I'll see if it looks feasible so then we can drop an extra dependency.